lib/index.js

```diff
@@ -0,0 +1,122 @@
+module.exports = {
```

Comment:
@e1himself do we need any rules from the [Slate ESLint config](https://github.com/prezly/slate/blob/master/.eslintrc.cjs) ported over here?

Reply:
Maybe `@typescript-eslint/consistent-type-imports`? 
With [Typescript 4.5 type import modifier](https://devblogs.microsoft.com/typescript/announcing-typescript-4-5/#type-on-import-names) it shouldn't cause pains.

Also, we went as far as to forbidding default exports in `@prezly/slate` with `import/no-default-export` altogether, as this is very often causing interoperability issues between ESM and CommonJS code. We don't see them in the app because we transpile everything with Webpack, which is has additional runtime hacks to hide the interoperability problems. Also with named exports it's just better DX (no need to rename default exports in imports and re-exports, IDE helps with importing and renaming refactoring is easier).

Reply:
Yeah, named exports is something we've been pushing to our team's repos recently as well, so it makes sense to make it consistent.

> Maybe @typescript-eslint/consistent-type-imports?
With Typescript 4.5 type import modifier it shouldn't cause pains.

This looks great!

Reply:
Added in 288666735a8e358677a615be88c3f8317af3a2b5